core/src/main/java/com/netflix/iceberg/BaseTableScan.java

```diff
@@ -231,4 +234,69 @@ public String toString() {
         .add("filter", rowFilter)
         .toString();
   }
+
+  private CloseableIterable<FileScanTask> planSplits(long splitSize) {
+    final CloseableIterable<FileScanTask> fileScanTasks = planFiles();
```

Comment:
Nit: no need to add `final` everywhere. This is automatically detected these days.

Reply:
I didn't know that. Will update